src/features/newProtectedData/NewProtectedData.tsx

```diff
@@ -111,15 +112,17 @@ export default function NewProtectedData() {
       )}
       {dataType === 'file' && (
         <Button
-          variant="contained"
-          component="label"
-          id="file"
-          fullWidth
-          onChange={handleFileChange}
-          sx={{ mt: 3 }}
+          className="uploadFileButton"
+          onClick={() => fileInput.current?.click()}
         >
           {!filePath ? 'Upload' : 'Updated File'}
-          <input hidden multiple type="file" />
+          <input
+            ref={fileInput}
+            hidden
+            multiple
+            type="file"
+            onChange={(e) => handleFileChange(e)}
+          />
         </Button>
```

Comment:
the upload button looks a bit aggressive maybe we should use the 'secondary' variant

| current | proposed |
| --- | --- |
| ![image](https://github.com/iExecBlockchainComputing/web3mail-usecase-demo/assets/26487010/3ad44b32-896c-4c57-b5ac-98cdf4350584) | ![image](https://github.com/iExecBlockchainComputing/web3mail-usecase-demo/assets/26487010/3d9fd809-6253-4e99-ad8f-ab5f755fdfea) |

Reply:
Commit: 343ef19e4dbccc3615f72e826e8f06e5ef71d4c6

Reply:
goog idear